docs/guides/general/remote-access.rst

```diff
@@ -11,303 +11,262 @@ your boot environments reside in encrypted filesystems, SSH access is necessary
+          mkdir -p /etc/zfsbootmenu/initcpio/{install,hooks}
+          cp /tmp/mkinitcpio-rclocal-master/rclocal_hook /etc/zfsbootmenu/initcpio/hooks/rclocal
+          cp /tmp/mkinitcpio-rclocal-master/rclocal_install /etc/zfsbootmenu/initcpio/install/rclocal
+          rm -r /tmp/mkinitcpio-rclocal-master
 
-  In the examples below, a couple of mkinitcpio modules will be installed to ``/etc/zfsbootmenu/initcpio`` to keep them
-  isolated from system-installed modules. To accommodate this non-standard installation, ``InitCPIOHookDirs`` must be
-  defined in ``/etc/zfsbootmenu/config.yaml``. Furthermore, because overriding the hook directory causes mkinitcpio to
-  ignore its default module path, the default ``/usr/lib/initcpio`` must be manually specified. If all hooks are
-  installed in ``/usr/lib/initcpio`` or ``/etc/initcpio``, the ZFSBootMenu configuration does **not** need to specify
-  ``InitCPIOHookDirs``.
+        Next, create an ``rc.local`` script that can be run within the mkinitcpio image to configure the ``eth0`` interface::
 
-Without further changes, running ``generate-zbm`` should now produce a ZBM image based on mkinitcpio rather than dracut,
-although it will lack networking and remote-access capabilities. (By default, ``generate-zbm`` instructs mkinitcpio to
-use the configuration at ``/etc/zfsbootmenu/config.yaml``, although this can be changed in the ``generate-zbm``
-configuration file.) For these features, some additional mkinitcpio modules and configuration changes are necessary.
+          cat > /etc/zfsbootmenu/initcpio/rc.local <<RCEOF
+          #!/bin/sh
 
-Because further configuration will require additional mkinitcpio modules, and these must be run before the
-``zfsbootmenu`` module in the initramfs, edit ``/etc/zfsbootmenu/mkinitcpio.conf`` and **remove** any ``zfsbootmenu``
-entry in the ``HOOKS`` definition. As the standard configuration file notes, the ``zfsbootmenu`` module is required for
-ZFSBootMenu to function, but ``generate-zbm`` will forcefully at this at the end of the module list. Thus, the simplest
-way to ensure that additions to the ``HOOKS`` array occur *before* the ``zfsbootmenu`` module is to omit the latter from
-the configuration. The standard ``HOOKS`` line in ``/etc/zfsbootmenu/mkinitcpio.conf`` should therefore be something
-like::
+          # Don't attempt to configure an interface that does not exist
+          ip link show dev eth0 >/dev/null 2>&1 || exit
 
-  HOOKS=(base udev autodetect modconf block filesystems keyboard)
+          # Bring up the interface
+          ip link set dev eth0 up
 
-.. _remote-mkinitcpio-net:
+          # Configure a static address for this host
+          ip addr add 192.168.1.2/24 brd + dev eth0
+          ip route add default via 192.168.1.1
 
-Basic Network Access
-~~~~~~~~~~~~~~~~~~~~
+          # Add some name servers
+          cat > /etc/resolv.conf <<-EOF
+            nameserver 1.1.1.1
+            nameserver 8.8.8.8
+            EOF
```

Comment:
```suggestion
          cat > /etc/resolv.conf <<-EOF
          	nameserver 1.1.1.1
          	nameserver 8.8.8.8
          	EOF
```
Not sure if Sphinx will preserve a tab inside this block, but I think the shell will refuse to recognize the indented heredoc unless real tabs are used. That could bite people copying and pasting this block if we can't include tabs.

Reply:
it's not preserved, I think we can just use a `<<` heredoc instead